### ISSUE TITLE
nemesis_seed can be undefined 

### ISSUE THREAD

**Issue by fruch** [scylladb/scylla-cluster-tests#5499]
```
[2022-11-28T17:31:28.988Z] if nemesis_seed := self.tester.params["nemesis_seed"]:
[2022-11-28T17:31:28.988Z] KeyError: 'nemesis_seed'
```

**Comment by fruch** [comment]
> WDYT, should it be `if nemesis_seed := self.tester.params["nemesis_seed"] else random.randint(1, 500):` ? in this case, we will still be able to reproduce issues when needed, and we solve the problem when a `nemesis_seed` is not set

could be, but the issue here is that it should be `self.tester.params.get("nemesis_seed")`

as to the logic of selecting a seed, this a different story

**Comment by fruch** [comment]
Anyhow this was introduce by #4655, so @aleksbykov and @roydahan, might want to comment on the logic of the seed usage.

Also I would recommend setting the seed as close as possible to the random calls (otherwise there could me more random calls, that will change the result), or use a specific Random object for the specific random calls intended.

**Comment by roydahan** [comment]
The assumption @aleksbykov does is incorrect.
Since if we don't have seed it randomize one.

**Comment by fruch** [comment]
fixed in 9d8ceb7067ee75f57523bcf97aabaf30080694b8
